staff/PacoGD/blog/index.html

```diff
@@ -0,0 +1,89 @@
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="icon" href="images/crossfit.jpg" type="icon/jpg" sizes="16x16">
+    <title>Crossfit Blog</title>
+</head>
+
+<body>
+
+    <header>
+        <!-- Cabecera del html, puede ser un menu con botones tambien-->
+        <h1>Blog CrossfitSkylab</h1>
+    </header>
+
+    <main>
+        <article>
+            <header>
+                <h2>Ejercicios para ponerte en forma</h2>
+                <time datetime="2014-10-05">Hace 3 días</time>
+            </header>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nulla sapiente impedit in at omnis, necessitatibus temporibus
+                expedita magnam, itaque maiores facere id ullam ratione, deserunt sed! Unde laboriosam perspiciatis doloribus.
+                Lorem ipsum, dolor sit amet consectetur adipisicing elit. Nobis, beatae, officia voluptatibus rerum magni,
+                asperiores deserunt quasi tenetur modi rem consequuntur aut exercitationem temporibus. Vitae nulla eveniet
+                maiores cumque. Aliquid. Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam unde, odit nisi perferendis
+                quo sit illum facere fugit corrupti. Nam eligendi a animi veniam nemo repudiandae quae nisi tempore ipsa!</p>
+
+            <footer>
+                <a href="links/index.html">Marco Garcia</a>
+                </footer>
+        </article>
+        <article>
+            <header>
+                <a href="https://www.muyinteresante.es/salud/fotos/alimentos-con-muchas-vitaminas" target="_blank">
+                    <h2>Alimentación en el deporte</h2>
+                </a>
+
+                <time datetime="2014-10-01">Hace 1 semana</time>
+            </header>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nulla sapiente impedit in at omnis, necessitatibus temporibus
+                expedita magnam, itaque maiores facere id ullam ratione, deserunt sed! Unde laboriosam perspiciatis doloribus.
+                Lorem ipsum, dolor sit amet consectetur adipisicing elit. Nobis, beatae, officia voluptatibus rerum magni,
+                asperiores deserunt quasi tenetur modi rem consequuntur aut exercitationem temporibus. Vitae nulla eveniet
+                maiores cumque. Aliquid. Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam unde, odit nisi perferendis
+                quo sit illum facere fugit corrupti. Nam eligendi a animi veniam nemo repudiandae quae nisi tempore ipsa!</p>
+            <footer>
+                Nacho Soto
+            </footer>
+        </article>
+        <nav>  <a href="https://www.muyinteresante.es/salud/fotos/alimentos-con-muchas-vitaminas" target="_blank">
+            <h2>Links páginas</h2>
+        </a>
+            <!-- Es donde pondremos los links relacionados con el tema de navegación, en un blog por ejemplo serian los últimos artículos relacionados con el tema-->
+        </nav>
```

Comment:
Usually this kind of link section is inside an < aside > tag.

Reply:
Done!